_gsocblogs/2022/blog_IRIS-HEP_KushKothari.md

```diff
@@ -2,7 +2,7 @@
 project: IRIS-HEP
 title: "Uproot + Dask: Blog 1"
```

Comment:
Please consider the recommendations for the final blogs sent in this [message](https://groups.google.com/g/cern-hsf-gsoc-students-2022/c/w_fIfhDdcG8/m/Av5ceK0VCAAJ). The title should rather match your project name. For the intro, you could just use the first two sentences from your actual introduction. If no photo, just use your GitHub avatar link. All this is NOT mandatory, if you chose not to do it, just mark this as resolved.

Reply:
Made the suggested changes. Thank you